Libs/Gst/Classes/GhostPad.cs

```diff
@@ -12,7 +12,7 @@ public partial class GhostPad
         // which *must* be run. Therefore our existing constructor design is
         // inadequate.
         public static GhostPad New(string name, Pad target)
-            => WrapHandle<GhostPad>(Native.@new(name, target.Handle));
+            => WrapHandle<GhostPad>(Native.@new(name, target.Handle), false);
```

Comment:
Should this be `WrapNullableHandle` as constructor functions can fail? Although, we assume that object creation always succeeds when using normal constructors, so perhaps not. Thoughts?

Reply:
I think it does not matter a lot currently especially as it is not the constructor but some helper method. Perhaps we need to put a bit more thought in it if we generate methods like this later. 

I updated it, just to match the documented behavior as this is the only argument I currently have.